### PR TITLE
Bug - Fix i18n experience headings

### LINKED ISSUE #4555: Bug - String missing i18n
## Description
The string **experiences** is not available for i18n. The title prop values for the `ExperienceByType` component should also be changed to include the full string (ex. **Education experiences** instead of just **Education**) since the strings in French will not fit the format currently used `{title} experiences`.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/66a95cffb5c7a8ffb27a7ba82a775cf255a7054d/frontend/common/src/components/UserProfile/ExperienceByTypeListing.tsx#L44

## Screenshot
Screen Shot 2022-11-02 at 11 40 28

## Acceptance Criteria
- [ ] appropriate values for title prop for the `ExperienceByType` component
- [ ] French strings should be translated appropriately

### PR DESCRIPTION
**Pull request by mnigh**
Resolves #4555.

## Note
Sent to translation:

```
{
  defaultMessage: "Personal experiences",
  id: "6VyRZ/",
  description:
  "Heading for personal experiences in experience by type listing",
},
{
  defaultMessage: "Community experiences",
  id: "iWD2Pz",
  description:
  "Heading for community experiences in experience by type listing",
},
{
  defaultMessage: "Work experiences",
  id: "aBSEkP",
  description:
  "Heading for personal experiences in experience by type listing",
},
{
  defaultMessage: "Education experiences",
  id: "pV96Xv",
  description:
  "Heading for education experiences in experience by type listing",
},
{
  defaultMessage: "Award experiences",
  id: "X0YPib",
  description:
  "Heading for award experiences in experience by type listing",
}
```

### CODE REVIEW

**Comment by mnigh** [comment]
Received from translation:

```
{
  defaultMessage: "Expériences personnelles",
  id: "6VyRZ/",
  description:
  "Heading for personal experiences in experience by type listing",
 },
 {
  defaultMessage: "Expériences de la collectivité",
  id: "iWD2Pz",
  description:
  "Heading for community experiences in experience by type listing",
 },
 {
  defaultMessage: "Expériences de travail",
  id: "aBSEkP",
  description:
  "Heading for personal experiences in experience by type listing",
 },
 {
  defaultMessage: "Expériences scolaires",
  id: "pV96Xv",
  description:
  "Heading for education experiences in experience by type listing",
 },
 {
  defaultMessage: "Expériences relatives aux prix",
  id: "X0YPib",
  description:
  "Heading for award experiences in experience by type listing",
 }
```

File: frontend/common/src/components/UserProfile/ExperienceByTypeListing.tsx [hunk 1]
```diff
@@ -93,8 +93,10 @@ const ExperienceByTypeListing: React.FunctionComponent<
       {personalExperiences.length > 0 ? (
         <ExperienceByType
           title={intl.formatMessage({
-            defaultMessage: "Personal",
-            id: "NDx+B0",
```
Comment (vd1992) [inline frontend/common/src/components/UserProfile/ExperienceByTypeListing.tsx:97]: I suppose to go the extra mile you could delete these from Fr.json 🤔 
Unless there is a reason to keep them around (potentially need the translated words)

**Review by vd1992 — APPROVED** [review]
looks good!

image

image

File: frontend/common/src/components/UserProfile/ExperienceByTypeListing.tsx (hunk 1 above)
Comment (mnigh) [reply frontend/common/src/components/UserProfile/ExperienceByTypeListing.tsx:97]: Ya, I thought about this too (and have before https://github.com/GCTC-NTGC/gc-digital-talent/issues/4489). The weird part about these strings is that one of them, Education `jtygmI`, is used elsewhere.

Comment (vd1992) [reply frontend/common/src/components/UserProfile/ExperienceByTypeListing.tsx:97]: ah just one of them, interesting, better safe than sorry

**Pull request merged**